js/test_util.ts

```diff
@@ -55,10 +52,40 @@ function permFromString(s: string): DenoPermissions {
   };
 }
 
+const processPerms = Deno.permissions();
+
+function permissionsMatch(
+  processPerms: Deno.Permissions,
+  requiredPerms: DenoPermissions
+): boolean {
+  for (const permName in processPerms) {
+    // if process has permission enabled and test case doesn't need this
+    // perm then skip
+    if (processPerms[permName] && !requiredPerms.hasOwnProperty(permName)) {
```

Comment:
if(`processPerms[permName]` <= this will always be true

Reply:
Not really if you run `deno --allow-net` only `processPerms["net"]` will be true, other permissions default to false.

Reply:
oh sorry i was admitting you were testing if we value was set or not. Sorry.